Shared/Components/ImageView.swift

```diff
@@ -79,7 +85,7 @@ extension ImageView {
             sources: [source].compacted(using: \.url),
             image: { $0 },
             placeholder: nil,
-            failure: { EmptyView() }
+            failure: { DefaultFailureView() }
```

Comment:
Need to remove the changes to these inits as well.

Reply:
done